rust/private/rustc.bzl

```diff
@@ -1029,6 +1038,30 @@ def construct_arguments(
 
     return args, env
 
+def _get_panic_style(ctx, toolchain):
+    panic_style = "unwind"
+
+    # Most targets default to unwind, but a few default to abort. Can't find a list in the
+    # documentation, this list is extracted from `library/panic_unwind/src/lib.rs` as of 1.68.1.
+    target_triple = toolchain.target_triple
+    if target_triple.arch.startswith("wasm") or target_triple.arch == "avr" or target_triple.system in ("none", "uefi", "espidf"):
```

Comment:
Could we move this logic over to `toolchain.bzl`, and use a `toolchain.default_panic_style` here?

Reply:
Done